src/Rules/Library/BoundingRectangleSizeReasonable.cs

```diff
@@ -36,7 +36,8 @@ protected override Condition CreateCondition()
             return IsNotOffScreen
                 & BoundingRectangle.NotNull
                 & BoundingRectangle.CorrectDataFormat
-                & ~ignoreableText;
+                & ~ignoreableText
+                & ~BoundingRectangle.TelerikSparklineItemstatusContext;
```

Comment:
Can we leverage `PropertyCOnditions.WPF` here, as opposed to burying it in the BoundingRectangle Concition? I _think_ this would be
```suggestion
                & ~(WPF & BoundingRectangle.TelerikSparklineItemstatusContext);
```